db/2016_legislators.csv

```diff
@@ -0,0 +1,1428 @@
+crp,lastname,firstname,party,distIdRunFor,fec_id
```

Comment:
You'll typically not upload these files to GH in a production environment. They're pretty large files, and now that you have them in the DB, you don't need them here on GH.